src/detections/chart/hooks/use-message-props.js

```diff
@@ -0,0 +1,10 @@
+import { useStyles } from 'detections/chart/style'
+
+export const useMessageProps = () => (
+  {
```

Comment:
...why? :smile:

Reply:
![image](https://user-images.githubusercontent.com/23569938/129173315-31234995-7595-4f6c-992f-34f9844adefe.png)
Let's add the `container` props here too.

Reply:
done